v1/api/swagger/swagger.yaml

```
@@ -177,3 +181,17 @@ components:
       type: apiKey
       in: header
       name: Membership
+    Role:
+      description: >-
+        The GLP IAM role. Required only when using GreenLake Platform IAM issued JWT for
+        authentication
+      type: apiKey
+      in: header
+      name: X-Role
+    Workspace:
+      description: >-
+        The GLP IAM workspace ID. Required only when using GreenLake Platform IAM issued JWT for
+        authentication
+      type: apiKey
+      in: header
+      name: X-WorkspaceId
```

Not sure if this is case sensitive, but better safe... Other instances are generated, I assume.
```suggestion
      name: X-WorkspaceID
```

---

Changed to X-Workspaceid as per canonical format.